### PR TITLE
Social book share

### PR DESCRIPTION
**Pull request by NguyenThiHangB**

### CODE REVIEW

**Comment by NguyenThiHangB** [comment]
ready
Link task: https://dev.framgia.com/issues/187244

**Comment by hungnh103** [comment]
commented
3 commit rồi em ơi

**Comment by NguyenThiHangB** [comment]
ready

**Comment by mdmahmudurrahman** [comment]
commented
Em, please resolve the conflict nhé

File: app/models/book.rb [hunk 1]
```diff
@@ -30,8 +30,9 @@ class Book < ApplicationRecord
       .group(:id).order("count(books.id) desc").limit(4)
   end)
   scope :reading_max, (lambda do |status|
-    joins(:mark_books).select(:id, :image, :name, :price, "count(books.id) as reading")
-      .where("mark_books.status": status).group(:book_id).order("count(books.id) desc").limit(4)
+    joins(:mark_books).select(:id, :image, :name, :price,
+    "count(books.id) as reading").where("mark_books.status": status)
+    .group(:book_id).order("count(books.id) desc").limit(4)
```
Comment (huongnguyenmta) [inline app/models/book.rb]: khai báo `4` trong settings đi em

**Comment by huongnguyenmta** [comment]
conflicted
con phờ líc kìa gái (??) =))

**Comment by NguyenThiHangB** [comment]
ready

**Pull request merged**